apps/labs/posts/pydata-sparse-maintenance-and-docs-overhaul.md

```diff
@@ -0,0 +1,177 @@
+
+- The `mkdocs-jupyter` plugin, which lets python files to be run and rendered as notebooks on the site. These python files were examples that already existed in the repository but were not being run nor shown in the previous website.
+- The `KaTex` library with the pymdonx.arithmatex markdown extension permits LaTeX mathematical formulas to be rendered.
+- To add some style to the main theme I added extra `css`. I wanted the site to look better from the default theme to improve the user experience. I also included card grid and the logo to the home page.
+
+<figure>
+    <img 
+    src="/posts/pydata-sparse-maintenance-and-docs-overhaul/screen.png"
+         alt="Home page image with a top menu and a grid with 6 cards"
+         />
+    <figcaption>Resulting Home page hovering over the Introduction card</figcaption> 
+</figure>
+
+All the dependencies that I added for the documentation are included in the `pyproject.toml` file. The plugins, extensions and settings are in the `mkdocs.yml` file.
+
+### Deployment to Read the Docs
+
+`Material for MkDocs` is designed to be deployed on `GitHub Pages`. It works "out-of-the box" with it. So, deploying it to [`ReadtheDocs`](https://about.readthedocs.com/) which is a documentation host, took some experimenting. But keeping the site there, meant that the previous releases (done with Sphinx) would still be available.
+
+The first step to deploy to `Read the Docs` is to create an account and authorize its access to the `GitHub` account. The authorization means that `Read the Docs` will be able to install webhooks to the `GitHub` repository.
+
+Normally, the project must be imported to `Read the Docs`. Since the old site was already deployed to it, I didn't have to do this step for `sparse`. But to make sure I understood the whole process, I first deployed my own fork of `sparse`. This turned out to be very useful later.
+
+I was given "Admin" rights to `sparse` in the host (`Read the Docs`), and I activated the "trigger builds from pull requests". This is the feature that lets us preview the documentation builds with each commit in `GitHub` with a link.
+
+The tricky part was that Hameer, the owner of the project on `GitHub`, also needed to link his `Read the Docs` account to `GitHub`.
+
+If I had started by deploying `sparse` directly, without installing my own fork first, it would have been impossible to find the missing part. The reason being that I didn't have access to all the project settings on `GitHub`. But I knew what was needed: to connect both accounts (`GitHub` and `Read the Docs`). This part was a good troubleshooting exercise.
+
+At the time of writing, the new website is available only under the "latest" version on `Read the Docs`. The stable version is still the `Sphinx` website, as the next release is still not ready. But it will be part of the next release.
+
+The resulting documentation site can be seen [here.](https://sparse-nd--747.org.readthedocs.build/en/latest/)
```

Comment:
Seems like you corrected the last part of the URL but forgot the first part. 😅 

```suggestion
The resulting documentation site can be seen [here.](https://sparse.pydata.org/en/latest/)
```

Reply:
Yes I saw it too late!